metadata/build.gradle

```
@@ -0,0 +1,77 @@
+
+def platforms = rootProject.enabled_platforms.split(',')
+def variants = rootProject.build_variants.split(',')
+
+base {
+    archivesName = "metadata"
+    libsDirectory = layout.buildDirectory.dir("libs")
+}
+
+// For each platform/variant combination, provide a jar containing the generated metadata.
+// This allows us to include the metadata files as a dependency, which is more flexible.
+platforms.each { platform -> variants.each { variant ->
+
+    def name = platform
+    def classifier = platform
+    def taskDescription = "Generate metadata for ${platform.capitalize()}"
+
+    if (variant != "normal") {
+        name += variant.capitalize()
+        classifier += '-' + variant
+        taskDescription += " (${variant.capitalize()})"
+    }
+
+    def task = tasks.register("${name}Jar", Jar) {
+        // Configure the task
+        group = "build"
+        description = taskDescription
+
+        archiveClassifier = classifier
+
+        // Include files under `src/platform`
+        from(layout.projectDirectory.dir("src").dir(platform))
+
+        inputs.properties(
+                mod_id: rootProject.name,
+                modrinth_name: project.modrinth_name,
+                version: project.mod_version,
```

Should the version shown in-game (and defined in mod metadata) be the `mod_version` or the _full_ version? IIRC it was previously the full version (including the mc version.
```suggestion
                version: project.version,
```

---

Yes, I think mod_version is better. Not sure why I had it using the full version.